src/Engine/ProtoImperative/Properties/Resources.resx

```diff
@@ -132,6 +132,9 @@
   <data name="FunctionPointerNotAllowedAtBinaryExpression" xml:space="preserve">
     <value>Function pointer is not allowed at binary expression other than assignment!</value>
   </data>
+  <data name="ImperativeSymbolsAreReadOnly" xml:space="preserve">
+    <value>Variables declared in outer scope are read-only inside imperative block!</value>
+  </data>
```

Comment:
I don't mean to bikeshed it, but I think this is a bit more clear. You might ask @kronz or @Racel for thoughts:

`Variables declared in an outer scope cannot be changed inside an imperative block.`

Reply:
Yes, the word `read-only` may be too technical for our audience. `bikeshed` is definitely for me :)